src/components/StructuredList/StructuredList-story.js

```diff
@@ -86,39 +84,23 @@ storiesOf('StructuredList', module)
               defaultChecked={!i || null}
             />
             <StructuredListCell>
-              <Icon
+              <CheckmarkFilled16
                 className="bx--structured-list-svg"
-                icon={iconCheckmarkSolid}
-                description="select an option"
-              />
+                aria-label="select an option">
+                <title>select an option</title>
+              </CheckmarkFilled16>
             </StructuredListCell>
-          </>
-        );
-        const structuredListBodyColumns = i => [
-          <StructuredListCell>Row {i}</StructuredListCell>,
-          <StructuredListCell>Row {i}</StructuredListCell>,
-          <StructuredListCell>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui
-            magna, finibus id tortor sed, aliquet bibendum augue. Aenean posuere
-            sem vel euismod dignissim. Nulla ut cursus dolor. Pellentesque
-            vulputate nisl a porttitor interdum.
-          </StructuredListCell>,
-        ];
-        return Array.apply(null, Array(numRows)).map((n, i) => (
-          <StructuredListRow label htmlFor={`row-${i}`}>
-            {componentsX
-              ? [...structuredListBodyColumns(i), checkbox(i)]
-              : [checkbox(i), ...structuredListBodyColumns(i)]}
           </StructuredListRow>
         ));
       };
       return (
         <StructuredListWrapper selection border>
           <StructuredListHead>
             <StructuredListRow head>
-              {componentsX
-                ? [...structuredListHeadColumns, emptyStructuredListHeadCell]
-                : [emptyStructuredListHeadCell, ...structuredListHeadColumns]}
+              <StructuredListCell head>ColumnA</StructuredListCell>,
+              <StructuredListCell head>ColumnB</StructuredListCell>,
+              <StructuredListCell head>ColumnC</StructuredListCell>,
```

Comment:
```suggestion
              <StructuredListCell head>ColumnC</StructuredListCell>
```

(same as L71-72 and L100-101) these commas are incorrectly being rendered